index.bs

```diff
@@ -130,6 +131,51 @@ This specification depends on the Infra Standard. [[!INFRA]]
 
 Network protocol messages are defined using CDDL. [[!RFC8610]]
 
+Tasks are scheduled using the <dfn>WebDriver task queue</dfn>, which is a
+the result of [=starting a new parallel queue=].
+
+To <dfn>queue a WebDriver task</dfn>, consisting of a list of steps, |steps|,
+[=enqueue steps=] |steps| to the [=WebDriver task queue=].
+
+This specification defines a <dfn>wait queue</dfn> which is a map.
+
+Issue: Surely there's a better mechanism for doing this "wait for an event" thing.
```

Comment:
This looks awfully familiar: https://w3c.github.io/webdriver-bidi/#infrastructure

Since the existing "wait for an event" thing still remains, is this PR still relevant in the current form? I see there are conflicts, not sure if the other mechanism was landed after this PR was sent...